.travis.yml

```diff
@@ -34,7 +34,7 @@ before_install:
         popd
         docker run -d -p 8020:8020 -p 50070:50070 -v $(pwd):/distributed distributed-hdfs
         export CONTAINER_ID=$(docker ps -l -q)
-        sleep 60  # Wait for namenode and datanode
+        sleep 10  # Wait for namenode and datanode
```

Comment:
Travis.ci can be very slow.  Even at 60s this is still the source of some intermittent failures.  In the future it would be nice to poll instead.